sdk/appconfiguration/app-configuration/test/auth.spec.ts

```diff
@@ -2,24 +2,34 @@
 // Licensed under the MIT license.
 
 import { AppConfigurationClient } from "../src";
-import { getTokenAuthenticationCredential, CredsAndEndpoint } from "./testHelpers";
+import { startRecorder, getTokenAuthenticationCredential, CredsAndEndpoint } from "./testHelpers";
 import * as assert from "assert";
+import { Recorder, env } from "@azure/test-utils-recorder";
 
-describe("Authentication", () => {
+describe.only("Authentication", () => {
   let credsAndEndpoint: CredsAndEndpoint;
+  let recorder: Recorder;
+  let uniqueId: string;
 
-  before(function() {
+  beforeEach(function() {
+    recorder = startRecorder(this);
+    uniqueId = recorder.getUniqueName("");
     credsAndEndpoint = getTokenAuthenticationCredential() || this.skip();
   });
 
+  afterEach(function() {
+    recorder.stop();
+  });
+
   it("invalid connection string gives a decent error message", () => {
     assert.throws(
       () => new AppConfigurationClient("an invalid connection string"),
       /Invalid connection string\. Valid connection strings should match the regex 'Endpoint=\(\.\*\);Id=\(\.\*\);Secret=\(\.\*\)'/
     );
   });
 
-  it("token authentication works", async () => {
+  // TODO: How do I make this work?
+  it.skip("token authentication works", async () => {
```

Comment:
@richardpark-msft @chradek help! I don't know how to run this test.

Reply:
Follow these instructions in the sample.env:

https://github.com/Azure/azure-sdk-for-js/blob/0e07e5ea3e5f9f98efd931374805b42ab6c73d0f/sdk/appconfiguration/app-configuration/sample.env

Reply:
I sent you the environment variables I'm using through Teams. They seem to  be matching that sample.env !

Reply:
Seems like my RBAC stuff wasn't fully propagated on Friday or something. Today, this test is passing 🌞